api/src/bigquerytest/java/org/pmiops/workbench/api/CohortBuilderControllerBQTest.java

```diff
@@ -575,6 +575,15 @@ private static SearchParameter wheelchair() {
         .standard(false);
   }
 
+  private static SearchParameter hasPMData() {
```

Comment:
should method be renamed to searchPMData or something like that? hasPMData sounds more like the method should return true and false rather than Object

Reply:
Done.